api/apistruct/struct.go

```diff
@@ -79,6 +79,7 @@ type CommonStruct struct {
 // FullNodeStruct implements API passing calls to user-provided function values.
 type FullNodeStruct struct {
 	CommonStruct
+	SentinelStruct
```

Comment:
Can avoid the sentinel.SentinelUnavailable thing in node builder with
```suggestion
	SentinelStruct `optional:"true"`
```